SPEC/PUBSUB.md

```diff
@@ -48,9 +48,11 @@ A great source of [examples][] can be found in the tests for this API.
 - `handler: (msg) => {}` - The handler to remove.
 - `callback: (Error) => {}` (Optional) Called once the unsubscribe is done.
 
-If no `callback` is passed, a [promise][] is returned.
+If a **handler is passed** and no `callback` is passed, a [promise][] is returned.
+
+If the `topic` and `handler` are provided, the `handler` will no longer receive updates for the `topic`. This behaves like `EventEmitter.removeListener`. If the `handler` is not equivalent to the `handler` provided on `subscribe`, no action will be taken.
 
-This works like `EventEmitter.removeListener`, as that only the `handler` passed to a `subscribe` call before is removed from listening. The underlying subscription will only be canceled once all listeners for a topic have been removed.
+If **only** the `topic` param is provided, unsubscribe will remove **all** handlers for the `topic`. This behaves like `EventEmitter.remoteAllListeners`. Use this if you would like to no longer receive any updates for the `topic`.
```

Comment:
Can we get a link to https://nodejs.org/dist/latest/docs/api/events.html#events_emitter_removealllisteners_eventname here also?

Reply:
done